visual-inspection/inspection-backend/inspection/model/model.py

```diff
@@ -99,5 +148,9 @@
 
 
 def decode_label(prediction):
-    original_label = tf.keras.applications.mobilenet_v2.decode_predictions(prediction, top=1)[0][0][1]
-    return GERMAN_LABELS[original_label] if original_label in GERMAN_LABELS else original_label
+    #original_list = tf.keras.applications.mobilenet_v2.decode_predictions(prediction, top=3)[0]
+    original_list = tf.keras.applications.xception.decode_predictions(prediction, top=3)[0]
+    for i in range(0,len(original_list)):
+        original_list[i] = list(original_list[i])
+        original_list[i][1] = GERMAN_LABELS[original_list[i][1]] if original_list[i][1] in GERMAN_LABELS else original_list[i][1]
```

Comment:
That is difficult to read, in particular, there is no need to use `range` here.

This slightly modified variant of the original code should do the same:

```suggestion
    return [GERMAN_LABELS[original_label[1]] if original_label[1] in GERMAN_LABELS else original_label[1] for original_label in original_list]
```